stdlib/zipfile.pyi

```diff
@@ -25,6 +25,11 @@ class _ZipStream(Protocol):
     # def tell(self) -> int: ...
     # def seek(self, __n: int) -> object: ...
 
+class _EndRecDataStream(Protocol):
```

Comment:
What does "EndRec" stand for?

Maybe `_SupportsReadSeekTell`.

Reply:
It's a class in zipfile that has these requirements. But I'll change it to a more standard name and add a comment instead.